src/result.js

```diff
@@ -111,6 +114,15 @@ class Result {
     };
   }
 
+  /**
+   * Get request specific analytics information.
+   *
+   * @returns {Object} analytics about the current request
+   */
+  getAnalytics() {
```

Comment:
I do not understand the point of this function, why do we need a `result.getFoo()` when we could use `result.foo` instead?

Reply:
We can do either, I was more inclined to putting a getter function but can do `result.foo` as well.